test/integration/ads_integration_test.cc

```diff
@@ -696,8 +778,9 @@ TEST_P(AdsIntegrationTest, ClusterWarmingOnNamedResponse) {
 
   // Send the second warming cluster.
   sendDiscoveryResponse<envoy::config::cluster::v3::Cluster>(
-      Config::TypeUrl::get().Cluster, {buildCluster("warming_cluster_2")},
-      {buildCluster("warming_cluster_2")}, {}, "3");
+      Config::TypeUrl::get().Cluster,
+      {buildCluster("warming_cluster_1"), buildCluster("warming_cluster_2")},
+      {buildCluster("warming_cluster_1"), buildCluster("warming_cluster_2")}, {}, "3");
```

Comment:
This is the correct CDS response when we want to send (additional) second warming cluster.